packages/hdwallet-keepkey/src/keepkey.ts

```diff
@@ -284,6 +284,10 @@ export class KeepKeyHDWallet implements HDWallet, BTCWallet, ETHWallet, DebugLin
     return (await this.getFeatures()).model;
   }
 
+  public async getFirmwareVersion(): Promise<string> {
+    return ""
```

Comment:
should pull this out of the features object:

`${features.majorVersion}.${features.minorVersion}.${features.patchVersion}`